[debug] [2025-10-10T00:15:20.338Z] ----------------------------------------------------------------------
[debug] [2025-10-10T00:15:20.342Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\HP\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login
[debug] [2025-10-10T00:15:20.342Z] CLI Version:   14.19.1
[debug] [2025-10-10T00:15:20.342Z] Platform:      win32
[debug] [2025-10-10T00:15:20.343Z] Node Version:  v22.17.1
[debug] [2025-10-10T00:15:20.343Z] Time:          Thu Oct 09 2025 18:15:20 GMT-0600 (hora estándar central)
[debug] [2025-10-10T00:15:20.343Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-10-10T00:15:20.345Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[info] i  The Firebase CLI’s MCP server feature can optionally make use of Gemini in Firebase. Learn more about Gemini in Firebase and how it uses your data: https://firebase.google.com/docs/gemini-in-firebase#how-gemini-in-firebase-uses-your-data 
[debug] [2025-10-10T00:15:20.786Z] <<< [apiv2][status] GET https://firebase-public.firebaseio.com/cli.json 200
[debug] [2025-10-10T00:15:20.786Z] <<< [apiv2][body] GET https://firebase-public.firebaseio.com/cli.json {"cloudBuildErrorAfter":1594252800000,"cloudBuildWarnAfter":1590019200000,"defaultNode10After":1594252800000,"minVersion":"3.0.5","node8DeploysDisabledAfter":1613390400000,"node8RuntimeDisabledAfter":1615809600000,"node8WarnAfter":1600128000000}
[debug] [2025-10-10T04:38:48.566Z] ExitPromptError: User force closed the prompt with SIGINT
    at Interface.sigint (C:\Users\HP\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\@inquirer\core\dist\commonjs\lib\create-prompt.js:101:37)
    at Interface.emit (node:events:518:28)
    at Interface.emit (node:domain:489:12)
    at [_ttyWrite] [as _ttyWrite] (node:internal/readline/interface:1132:18)
    at ReadStream.onkeypress (node:internal/readline/interface:271:20)
    at ReadStream.emit (node:events:530:35)
    at ReadStream.emit (node:domain:489:12)
    at emitKeys (node:internal/readline/utils:370:14)
    at emitKeys.next (<anonymous>)
    at ReadStream.onData (node:internal/readline/emitKeypressEvents:64:36)
[error] 
[error] Error: An unexpected error has occurred.
